src/ORM/Behavior/Translate/TranslateStrategyTrait.php

```diff
@@ -111,7 +111,7 @@ protected function unsetEmptyFields($entity)
         foreach ($translations as $locale => $translation) {
             $fields = $translation->extract($this->_config['fields'], false);
             foreach ($fields as $field => $value) {
-                if (strlen($value) === 0) {
+                if ($value === null || strlen($value) === 0) {
```

Comment:
Can we just use `!value`? It covers a couple other values, but they aren't really value translations either other than possibly "0".

Reply:
Having a string `'0'` is possibility so we could simply the check to `$value === null && $value === ''`.